packages/core-utils/src/query.js

```diff
@@ -209,3 +216,105 @@ export function planParamsToQuery(params, config) {
   });
   return query;
 }
+
+/**
+ * Create an object that can be used as a querystring in making an OTP
+ * PlannerResource request.
+ *
+ * See http://otp-docs.ibi-transit.com/api/resource_PlannerResource.html
+ *
+ * @param  {Object} config  The OTP application config. See types#configType
+ * @param  {Object} currentQuery  The current query parameters as saved in the
+ *   application state. This method does some extra logic on top of this data
+ *   in order to create a request suitable for OTP. See types#queryType
+ * @param  {boolean} ignoreRealtimeUpdates  If true, will create a request that
+ *   does not use realtime data.
+ */
+export function getRoutingParams(config, currentQuery, ignoreRealtimeUpdates) {
```

Comment:
If this is new code, where is it being used?

Reply:
It is for implementing applications to use. See #40.